Tasks/DotNetCoreCLIV2/task.json

```diff
@@ -501,14 +501,14 @@
         {
             "target": "feedRestore",
             "endpointId": "tfs:feed",
-            "endpointUrl": "{{endpoint.url}}/_apis/packaging/feeds?includeUrls=False",
+            "endpointUrl": "{{endpoint.url}}/_apis/packaging/feedids",
             "resultSelector": "jsonpath:$.value[*]",
             "resultTemplate": "{ \"Value\" : \"{{#if project}}{{{project.id}}}\\/{{/if}}{{{id}}}\", \"DisplayValue\" : \"{{{name}}}\" }"
```

Comment:
These `resultTemplate`s also need to be updated (project is not a separate object in the new API)

Reply:
fixed